client/src/components/PersonCard/PersonCard.js

```diff
@@ -3,8 +3,26 @@ import { Link } from "react-router-dom";
 import "./PersonCard.css";
 import ProfilePicture from "../ProfilePicture/ProfilePicture";
 import image from "../../assets/images/profile-picture-1.png";
+
 // Function based React Component
 const PersonCard = props => {
+  if (online) {
```

Comment:
Rather than a massive conditional like this where most of this is repeated, can we do a single conditional to show or hide the dot? Use the `&&` trick!